src/services/URLService.js

```diff
@@ -250,12 +269,17 @@ function urlService(baseUrl, mmsUrl) {
      */
     var getElementURL = function(reqOb) {
         var r = root + '/projects/' + reqOb.projectId + '/refs/' + reqOb.refId + '/elements/' + reqOb.elementId;
-        return addExtended(addTicket(addVersion(r, reqOb.commitId)), reqOb.extended);
+        return addExtended(addVersion(r, reqOb.commitId), reqOb.extended);
```

Comment:
as far as ve is concerned (for a cameo project), just changing the 'elements' path in url to 'views' should give the same results as before - the mms cameo module's added 'views' controller behaves the same way as 'elements' but will further process the ve specific _childViews key

Reply:
updated